.github/workflows/test.yml

```
@@ -11,7 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-13, windows-latest]
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.9", "3.11", "3.13"]
```

Lowest and highest version is good enough in my opinion.

```suggestion
        python-version: ["3.9", "3.13"]
```

---

I had 3.11 because it is the latest version that supports all solvers. But it would work with 3.9 but 3.11 pulls in much newer dependencies and I thought it might be good to test this.

---

Okay, that makes sense :slight_smile: